src/main/java/org/cirdles/topsoil/app/TSVTable.java

```diff
@@ -118,12 +140,14 @@ public void clear() {
         getColumns().clear();
     }
 
-    public void load() {
+    //Method used by TSVTableTest
+    public void load(Path savePath) {
         if (savePath != null) {
             loadFromPath(savePath);
         }
     }
 
+    //Method used by TSVTableTest
```

Comment:
Then refactor by rewriting the tests? I don't know if this comment is a note to self or a fix?